controls/pcidss_4.yml

```diff
@@ -2115,15 +2118,30 @@ controls:
       status: manual
 
     - id: 8.3.9
-      title: 'If passwords/passphrases are used as the only authentication factor for
-        user access (i.e., in any single-factor authentication implementation) then
-        either'
+      title: If passwords/passphrases are used as the only authentication factor for user access
+        (i.e., in any single-factor authentication implementation) they should have a limited
+        lifetime.
+      description: |-
+        If passwords/passphrases are used as the only authentication factor for user access (i.e.,
+        in any single-factor authentication implementation) then either:
+        - Passwords/passphrases are changed at least once every 90 days,
+        OR
+        - The security posture of accounts is dynamically analyzed, and real-time access to
+        resources is automatically determined accordingly.
       levels:
         - base
       status: automated
+      notes: |-
+        The requirement does not explicitily asks to define the number of days before the password
```

Comment:
/asks to/d

Reply:
Fixed in 5f6665a860ca87ae23caf173d48189366949121c . Thanks